WorldBank-Transport/DRIVER
# Release 1.2.2

## Overview

New bug fix release. See https://www.pivotaltracker.com/story/show/160102669 for details.

### Checklist


### Notes

The first three commits are the actual bug fixes we want to deploy. The last three commits are to make provisioning succeed.

## Testing Instructions

Let's get our local DRIVER instance into a similar state that it would be in on a server we're updating:

* Clone this repo again (`mkdir -p ~/git/test && cd ~/git/test && git clone git@github.com:WorldBank-Transport/DRIVER.git`)
* `touch gradle/data/driver.keystore`
* `cp deployment/ansible/group_vars/all.example deployment/ansible/group_vars/all`
* Check out master (`git checkout master`)
* Clone ashlar repo along side it (`cd ~/git/test && git clone git@github.com:azavea/ashlar.git`)
* Check out legacy tag (`git checkout legacy`)
* Back to DRIVER (`cd -`)
* Cherry pick the last three commits (see Notes) so that provisioning may succeed
* `vagrant up --provision`

Provisioning should have succeeded. Now we're in a state similar to an instance with the latest release.

* Get rid of those cherry picked commits (`git reset --hard HEAD~3`)
* Check out this branch
* `vagrant up --provision`

Now your environment should have the bug fixes. Load geographies and incidents as per the README instructions and test the (testable) fixes:

https://github.com/WorldBank-Transport/DRIVER/pull/711
https://github.com/WorldBank-Transport/DRIVER/pull/718

Closes #160102669
